src/middleware/jira-symmetric-jwt-middleware.ts

```diff
@@ -143,9 +145,16 @@ const checkPathValidity = (url: string) => {
 	});
 };
 
-const checkGenericContainerActionUrl = async (url: string) : Promise<boolean | undefined> => {
+export const checkGenericContainerActionUrl = async (url: string): Promise<boolean | undefined> => {
 	const genericContainerActionUrls = await getGenericContainerUrls();
-	return genericContainerActionUrls?.some(moduleUrl => {
-		return matchRouteWithPattern(moduleUrl, url);
-	});
+
+	const urls = genericContainerActionUrls
+		?.filter(moduleUrl => {
+			return !moduleUrl.includes("create-branch");
```

Comment:
I think this PR will work in the create branch context, hence approve it now.

But, I am curious in the generic contains context, 
1. Why `CREATE_BRANCH_ENDPOINT` should be included in `getGenericContainerUrls()` in the first place?
2. If the generic container somehow call that `CREATE_BRANCH_ENDPOINT` endpoint, what sort of token will it contains?